docs/upgrade/UPGRADE-v8.0.0.md

```diff
@@ -86,6 +86,20 @@ There you can find links to upgrade notes for other versions too.
         ```
     - check and fix your other tests, they might start failing if they assumed `Product::$availability` is not null when the product is using stock, or that stock quantity is not null when it's not using stock
 - follow upgrade instructions for entities simplification in the [separate article](./upgrade-instructions-for-entities-simplification.md) ([#1123](https://github.com/shopsys/shopsys/pull/1123))
+- the namespace of `FriendlyUrlDataProviderInterface` has changed from `Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\CompilerPass` to `Shopsys\FrameworkBundle\Component\Router\FriendlyUrl` so change all your usages accordingly ([#1140](https://github.com/shopsys/shopsys/pull/1140))
+    - the namespace of `FriendlyUrlDataProviderRegistry` has changed from `Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\CompilerPass` to `Shopsys\FrameworkBundle\Component\Router\FriendlyUrl` so change all your usages accordingly
+    - update your `service.yml` accordingly:
+    ```diff
+        services:
+            _defaults:
+                autowire: true
+        	        autoconfigure: true
```

Comment:
you have tabs here for some reason :slightly_smiling_face:

Reply:
I do not know how they got there... fixed